### ISSUE TITLE
Fix BDD Test Bug

### ISSUE THREAD

**Issue by silemie** [nyu-devops-fall19-suppliers/suppliers#153]
**As a** ...
**I need** ...
**So that** ...

**Assumptions:**
* ...

**Acceptance Criteria**
```
Given ...
When ...
Then ...
```

**Comment by lhmartin** [comment]
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/16001690/70383839-ffe5af00-1942-11ea-87fb-53374c4578ba.png">

This is the error currently stopping the site from deploying

**Comment by lhmartin** [comment]
<img width="595" alt="image" src="https://user-images.githubusercontent.com/16001690/70383873-7d112400-1943-11ea-9bec-45a23084c432.png">
It fails because there IS a supplier from walmart in there. @greenson Do you think this is because the db is now persistent and there was at some point a Walmart supplier added which is now messing up this test?

**Comment by lhmartin** [comment]
@silemie  @greenson @Krystal-158 I think then we should delete it from the database? Otherwise this test will never pass.

**Comment by Krystal-158** [comment]
Agreed, deleting that item is the fastest way. I guess the best way is to change "I should not see Walmart in the results" to "I should not see 3(or 2 or 1) in averageRating", but I have no idea how to implement this

**Comment by silemie** [comment]
Agreed. Or we can add one more method like "reset database"

**Comment by lhmartin** [comment]
Okay I will make a commit that will delete this entry, I don't think we should have something that resets the database within our service, unless we can protect it, otherwise anyone with an API key could reset the db.

**Comment by lhmartin** [comment]
Okay little update, now that we are using the swagger docs, a lot of our behave functionality is not working so now the site looks like: 

<img width="1534" alt="image" src="https://user-images.githubusercontent.com/16001690/70392273-5a672580-19ac-11ea-9ac4-4a1054bdc18f.png">

Meaning things like accessing search results are not done in the same way:
<img width="1426" alt="image" src="https://user-images.githubusercontent.com/16001690/70392279-6fdc4f80-19ac-11ea-9f0d-1c3b2340dce2.png">


So we will have to refactor our BDD steps to accommodate.

**Comment by silemie** [comment]
Yes. I discussed this issue with @Krystal-158. It seems like we should refactor our BDD code to fit swagger docs.
